Code/GraphMol/Wrap/Queries.cpp

```diff
@@ -144,6 +144,12 @@ Ret *PropQueryWithTol(const std::string &propname, const ExplicitBitVect &v,
   return res;
 }
 
+namespace {
+  Atom *replaceAtomWithQueryAtomHelper(ROMol &mol, Atom &atom) {
+    return QueryOps::replaceAtomWithQueryAtom(static_cast<RWMol *>(&mol), &atom);
```

Comment:
Might be useful to make a note that replaceAtomWithQueryAtom copies the atom.  I'm not sure this is actually documented anywhere.